docs/pages/guides/mesheryctl/mesheryctl_system.md

```diff
@@ -0,0 +1,124 @@
+---
```

Comment:
@alphaX86 @zakisk @abdullah1308 will this page be overwritten by Mesheryctl's self-publishing workflow? 
https://docs.meshery.io/reference/mesheryctl/system

Reply:
I think the guides doesn't comes under self-publishing workflow it is only for [mesheryctl command reference](https://docs.meshery.io/reference/mesheryctl/system)